Docker.DotNet/project.json

```diff
@@ -12,6 +12,7 @@
         "portable-dnxcore50+net45+win8+wp8+wpa81"
       ],
       "dependencies": {
+        "System.Buffers": "4.0.0-rc3-23829",
```

Comment:
I was having a lot of trouble getting it build, so I just used `rc3` for `netstandard`, and `4.0.0` for the rest. It would be nice if you could do it in another PR, thanks.